runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -101,7 +101,7 @@ Java_sun_misc_Unsafe_defineAnonymousClass(JNIEnv *env, jobject receiver, jclass
 	jsize length = env->GetArrayLength(bytecodes);
 
 	/* acquires access internally */
-	jclass anonClass = defineClassCommon(env, hostClassLoaderLocalRef, NULL,bytecodes, 0, length, protectionDomainLocalRef, J9_FINDCLASS_FLAG_UNSAFE | J9_FINDCLASS_FLAG_ANON, hostClazz);
+	jclass anonClass = defineClassAnonymous(env, hostClassLoaderLocalRef,bytecodes, length, protectionDomainLocalRef, J9_FINDCLASS_FLAG_UNSAFE | J9_FINDCLASS_FLAG_ANON, hostClazz, constPatches);
```

Comment:
If only the ramCP needs to be patched for Strings / Classes, can we handle it entirely here by calling a `patchRAMConstantPool(J9Class *clazz, jobjectArray cpPatch)`-style method here?